views/Profile.tsx

```diff
@@ -176,17 +176,35 @@ class Profile extends Component<Props> {
                       .slice(0, 3)
                       .map(badge =>
                         this.props.badgeOptions.includes(badge) ? (
-                          <Image
-                            key={badge}
-                            style={styles.ownBadge}
-                            source={badges[badge]}
-                          />
+                            <Tooltip 
+                              key={badge}
+                              popover={<Text style={{ fontWeight: '700', color: 'grey' }}>You're an event master!</Text>}
```

Comment:
The same styling is being applied to all tooltips, so you can abstract the styling into a class.